R/study_exclusions_panel.R

```diff
@@ -83,11 +83,37 @@ study_exclusions_panel_server <- function(id, data, treatment_df, reference_trea
       studies <- isolate(all_studies())
       return(studies[!studies %in% main_connected_data()$Study])
     })
+    
+    # Update the checkboxes to tick the excluded studies
+    observe({
+      # Select check boxes for disconnected studies
+      disconnected_indices <- which(all_studies() %in% main_subnetwork_exclusions())
+      selected = all_studies()[disconnected_indices]
+      shiny::updateCheckboxGroupInput(inputId = "exclusionbox", selected = selected)
+    }) |> bindEvent(main_subnetwork_exclusions())
 
-    # Update "selections" for the initial data to be those excluded by taking the subnetwork
+    # Update which studies can be selected from the sensitivity analysis by taking the initial data subnetwork
     observe({
-      selections(main_subnetwork_exclusions())
-    })
+      filtered_treatments <- FindAllTreatments(main_connected_data())
+      
+      lapply(
+        all_studies(),
+        function(study) {
+          index <- match(study, all_studies())
+          subElement <- glue::glue("#{session$ns('exclusionbox')} .checkbox:nth-child({index}) label")
```

Comment:
I assume this was named previously as it's camel case not snake case?

Reply:
Fixed. Old habits die hard...